### PR TITLE
Bump rubygem-get_process_mem to 0.2.7-1

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

**Comment by ehelms** [comment]
Can be built as an update now, will break under Ruby 2.7 with the current version.

File: packages/foreman/rubygem-get_process_mem/rubygem-get_process_mem.spec [hunk 1]
```diff
@@ -59,15 +65,16 @@ gem build %{gem_name}.gemspec
...
-cp -pa .%{gem_dir}/* \
+cp -a .%{gem_dir}/* \
         %{buildroot}%{gem_dir}/
 
 %files
 %dir %{gem_instdir}
+%{gem_instdir}/.github
+%exclude %{gem_instdir}/.gitignore
+%exclude %{gem_instdir}/.travis.yml
+%{gem_instdir}/appveyor.yml
```
Comment (ekohl) [inline packages/foreman/rubygem-get_process_mem/rubygem-get_process_mem.spec:76]: This can still be excluded since it's CI config.

**Review by ekohl — CHANGES_REQUESTED** [review]

File: packages/foreman/rubygem-get_process_mem/rubygem-get_process_mem.spec (hunk 1 above)
Comment (ehelms) [reply packages/foreman/rubygem-get_process_mem/rubygem-get_process_mem.spec]: To help ensure this systemically -- https://github.com/theforeman/foreman-packaging/pull/6414

**Review by pcreech — APPROVED** [review]
ACK

**Pull request merged**